source/crud/crud.md

```diff
@@ -2305,11 +2322,15 @@ unfortunate, we felt it better to have the bulk api be consistent with the rest
 operations. However, the fluent-bulk-api is still able to be used as this change is non-backwards breaking. Any driver
 which implemented the fluent bulk API should deprecate it and drivers that have not built it should not do so.
 
-Q: What about explain?\
-Explain has been determined to be not a normal use-case for a driver. We'd like users to use the
-shell for this purpose. However, explain is still possible from a driver. For find, it can be passed as a modifier.
-Aggregate can be run using a runCommand method passing the explain option. In addition, server 3.0 offers an explain
-command that can be run using a runCommand method.
+Q: Should drivers offer explain helpers?\
+Originally, it was determined that explain should not be exposed via
+specialized APIs in drivers (runCommand was always an option after server 3.0.). However, some drivers historically have
+offered explain APIs and continue to do so.
+
+Explain helpers are not required because it has been determined to be not a normal use-case for a driver. We'd like
+users to use the shell for this purpose. However, explain is still possible from a driver. For find, it can be passed as
+a modifier. Aggregate can be run using a runCommand method passing the explain option. In addition, server 3.0 offers an
+explain command that can be run using a runCommand method.
```

Comment:
> However, explain is still possible from a driver. For find, it can be passed as
a modifier. Aggregate can be run using a runCommand method passing the explain option. In addition, server 3.0 offers an
explain command that can be run using a runCommand method.

I find this section interesting it makes it sound like explain is an option that can be sent inside of a find/agg rather than wrapping it, but is that just a misinterpretation?

Reply:
I didn't write this section so I don't know for certain.  I think that prior to the explain command being added, explain was done via an option, not a separate command, and this is just a legacy explanation.

Reply:
I notice it mentions server 3.0. If this doesn't apply in any supported server versions is it worth removing?

Reply:
To clarify: server 3.0 introduced the [explain](https://www.mongodb.com/docs/manual/reference/command/explain/) command. Prior to that, queries were explained using the [$explain modifier](https://www.mongodb.com/docs/v5.0/reference/operator/meta/explain/). This all pre-dates the introduction of the `find` command.

[aggregate](https://www.mongodb.com/docs/manual/reference/command/aggregate/) has historically supported an `explain` option since 2.6, but its output is limited compared to the `explain` command. This is discussed in a bit more detail in [this old Google doc](https://docs.google.com/document/d/1kHIQLLZv7ZMlW9aktZ6CIQdC1L8j3_PJxBdUx_q3FcQ/edit#heading=h.ktd5lcont67g) on aggregation performance improvements.

I agree with removing any references to these old APIs from the CRUD spec.